File: ABC_083/b.js
// inputに入力データ全体が入る
function Main(input) {
  input = input.split(' ').map(n => parseInt(n, 10))
  var N = input[0]
  var A = input[1]
  var B = input[2]
  var ans = 0
  for(n=1;n<=N;n++){
    var sum = 0
    var tmp = n
    while(tmp>0){
      sum += tmp % 10
      tmp = Math.floor(tmp / 10)
    }
    if(sum >= A && sum <= B) {ans = ans + n}
  }
  console.log(ans)
}
//*この行以降は編集しないでください（標準入出力から一度に読み込み、Mainを呼び出します）
Main('20 2 5')
// Main(require("fs").readFileSync("/dev/stdin", "utf8"));

File: ABC_083/b.py
import math
N, A, B = list(map(int, input().split()))


def sum_of_digits(n):
    res = 0
    while n > 0:
        res += n % 10
        n = math.floor(n / 10)
    return res


total = 0
for i in range(1, N+1):
    s = sum_of_digits(i)
    if s >= A and s <= B:
        total += i
print(total)
